groupsforsections.py

```diff
@@ -149,6 +149,33 @@ def create_group(groups_for_section, group_category_id, group_name, course_id):
     return str(group['id'])
 
 
+def add_users_to_group(groups_for_section, group_id, user):
+    """
+    adding user to a group, parses Json, do exception handling
+    :param groups_for_section:
+    :param group_id:
+    :param user:
+    :type groups_for_section GroupsForSections
+    :type group_id str
+    :type user str
+    :return: membership_id
+    :rtype: int, None
+    """
+    logging.debug(add_users_to_group.__name__ + '() called')
+    try:
+        response = groups_for_section.add_users_to_group(group_id, user)
+
+    except requests.exceptions.RequestException as e:
```

Comment:
According to the documentation of the `requests` module all the exception that might happen is encompassed in the 'RequestException'. I think that should take care of it. 
http://docs.python-requests.org/en/master/user/quickstart/#errors-and-exceptions
https://raw.githubusercontent.com/kennethreitz/requests/master/requests/exceptions.py